README.md

```diff
@@ -131,6 +131,24 @@ Creates an Etch Packet and optionally sends it to the first signer.
       }
     ```
 
+##### getEtchPacket(options)
+
+Gets the details of an Etch Packet.
+* `options` (Object) - An object with the following structure:
+  * `variables` (Object) - Requires `eid`
+    * `eid` (String) - your Etch Packet eid
+  * `responseQuery` (String) - _optional_ A GraphQL Query compliant query to use for the data desired in the mutation response. Can be left out to use default.
```

Comment:
This is a `query` and not a `mutation`...so I think you should remove or change all references to `mutation`s.

Also, it doesn't look like you're allowing the User to override the entire query (the way that the `createEtchPacket` mutation allows it) with a `query` param, and that's totally fine and I think it should be that way...but then you should remove the mentioning of the `"responseQuery" param is ignored if "query" is passed` bits, since there is no `query` param.

Reply:
Woops, thought I got them all but missed a reference to mutation there. Also I'm going to leave out the query param since it's unnecessary.